### DISCUSSION TITLE
MapBoxGL error

### DISCUSSION THREAD

**Discussion by irvingbennett** [yihong0618/running_page#192] [General]
Hi yihong and company! I forked your running page and managed to get it running. It took me some work because it didn't want to build with Gatsby 2.x because some libraries were too old. I managed to get working with Gatsby 4.0 and its going great on my local machine! I really like your app.
I also managed to deploy the app on Vercel and Gatsby Cloud. Both those installations build and run (after some tweaks). Both of those installations fail to show the map and have MapBoxGL errors that can be seen in the developers console: 
`77e41395-87bb-4f6d-86fc-2386ddd3dbdd:1 An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel; please see https://docs.mapbox.com/mapbox-gl-js/guides/install/#transpiling`

Have you seen this problem in your deployments?

**Comment by irvingbennett** [comment]
By the way, this is the Vercel app: https://running-page-irving1.vercel.app/

**Comment by yihong0618** [comment]
let's try to fix this.
